product-scenarios/1-manage-users-and-roles/1.1-admin-user-manage-roles/1.1.2-manage-roles-with-scim1/src/test/resources/scim1-groups/scim1InternalGroup.json

```diff
@@ -0,0 +1,3 @@
+{
+  "displayName": "Internal/manager"
+}
```

Comment:
Add  new line and Please check this throughout the PR

Reply:
Added